mkosi/resources/mkosi-tools/mkosi.conf.d/10-debian-ubuntu/mkosi.conf.d/virtiofsd.conf

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=|!ubuntu
+Release=|!jammy
+
+[Match]
+Distribution=|!debian
+Release=|!bookworm
```

Comment:
This is is a bit hard to reason about
```
((not ubuntu) or (not jammy)) and ((not debian) or (not bookworm)) =>
(not (ubuntu and jammy)) and (not (debian and bookworm)) =>
not ((ubuntu and jammy) or (debian and bookworm)
```
so I think this is correct, but it could do with a comment, so that readers don't have to do the boolean logic :)